R/utilities.r

```diff
@@ -430,3 +430,21 @@ is_column_vec <- function(x) {
   dims <- dim(x)
   length(dims) == 2L && dims[[2]] == 1L
 }
+
+# Parse takes takes n strings and returns n expressions
```

Comment:
I noticed "takes takes". Since I asked for one more substantive change, maybe you could fix this also.